test/directory.test.ts

```diff
@@ -0,0 +1,347 @@
+import DeltaStorage, { DeltaStorageInit } from '../src'
+
+import crypto from 'crypto'
+
+import * as matchers from 'jest-extended'
+expect.extend(matchers)
+
+// NOTE: Replace this with your own API_KEY
+// TODO: Create a test account for each env
+const API_KEY =
+  '00000000-0000-0000-0000-000000000000.0.162eeecd-aac1-4a4d-89fd-97dc597f8f1e.7AQjso3c8ENenysfybUieZd3m3mBp2ZtDtpUBHMosPWn'
+
```

Comment:
gawan ko nalang to ng test env sunod since hassle magpalit palagi ng api key ahahaha ito kasi yung gusto ko palitan